### PR TITLE
Make page title configurable, make index a template

### PR DESCRIPTION
**Pull request by SeanMcGrath**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -163,3 +163,6 @@ dist
 
 # Generated python files for proto schemas
 app/jitsi/schema/*pb2.py
+
+# VSCode
+*.code-workspace
```
Comment (bMacSwigg) [inline .gitignore:168]: Nit: newline at end of file

File: app/client/styles/themes/_active.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+/Users/smcgrath/code/jitsi-party/app/client/styles/themes/_tst.scss
```
Comment (bMacSwigg) [inline app/client/styles/themes/_active.scss:1]: Can this be a relative path?

File: app/config/schema/config.proto [hunk 3]
```diff
@@ -13,9 +13,14 @@ message Config {
     // Optional. Defaults to false.
     bool debug = 1;
 
+    // Title for main page
+    // document.title in the DOM
+    // Required.
+    string pageTitle = 18;
```
Comment (bMacSwigg) [inline app/config/schema/config.proto:19]: Super nit: this should technically be `page_title` according to proto style. The json parser will still accept `pageTitle` in the config files.

**Review by bMacSwigg — APPROVED** [review]

File: app/client/styles/themes/_active.scss (hunk 2 above)
Comment (SeanMcGrath) [reply app/client/styles/themes/_active.scss:1]: oh this should be gitignored

**Pull request merged**